extensions/hibernate-envers/runtime/src/main/java/io/quarkus/hibernate/envers/HibernateEnversBuildTimeConfig.java

```diff
@@ -0,0 +1,13 @@
+package io.quarkus.hibernate.envers;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot
```

Comment:
If you extracted the config in the build step and just passed a boolean then it would be only used at build time. But that's not what you do. The recorder code is executed at runtime.